templates/endpoint_policies/s3_endpoint_policy.json.tpl

```diff
@@ -0,0 +1,115 @@
+{
+  "Statement": [
+    {
+      "Sid": "restrict-access-dirty-bucket-download",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "s3:GetObject",
+      "Resource": "arn:aws:s3:::tdr-upload-files-cloudfront-dirty-intg/*",
```

Comment:
Should the "intg" be hardcoded?

Reply:
Nope, I've replaced it with an environment variable.